### PR TITLE
(948) Add in screens for eligibility screens if a person doesnt meet the tier model

### PR DESCRIPTION
**Pull request by pezholio**
This adds an extra flow to the start of the Apply journey to check an offender's tier before continuing. If they do not meet the requirements, they're prompted to confirm if it's an exceptional case, and if so, add the details before continuing.

If they answer `No`, then the journey ends.

# Screenshots

Apply -- allows the user to specify if the case is exceptional if the offender's tier is not eligible

Apply -- allows the user to specify if the case is exceptional if the offender's tier is not eligible (1)

Apply -- tells the user that their application is not applicable if the tier is not eligible and it is not an exceptional case

Apply -- tells the user that their application is not applicable if the tier is not eligible and it is not an exceptional case (1)

### CODE REVIEW

**Comment by pezholio** [comment]
[Add in screens for elegibility screens (if a person doesn't meet the TIER model)](https://trello.com/c/wB28GkJC/948-add-in-screens-for-elegibility-screens-if-a-person-doesnt-meet-the-tier-model)

**Comment by pezholio** [comment]
[Add in screens for elegibility screens (if a person doesn't meet the TIER model)](https://trello.com/c/wB28GkJC/948-add-in-screens-for-elegibility-screens-if-a-person-doesnt-meet-the-tier-model)

File: server/form-pages/apply/reasons-for-placement/basic-information/exceptionDetails.ts [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+
+    return response
+  }
+
+  previous() {
+    return 'is-exceptional-case'
+  }
+
+  next() {
+    return 'sentence-type'
```
Comment (bryangaledxw) [inline server/form-pages/apply/reasons-for-placement/basic-information/exceptionDetails.ts:73]: Is any work needed on the `previous()` method for the sentence-type page, to conditionally go back to the exception details page?

**Review by bryangaledxw — APPROVED** [review]
Small typo in commit message of "Add an isApplicableTier util method" = "is the sex" -> "if the sex"

Otherwise, LGTM, just a few minor bits of feedback

File: server/form-pages/apply/reasons-for-placement/basic-information/exceptionDetails.ts (hunk 1 above)
Comment (pezholio) [reply server/form-pages/apply/reasons-for-placement/basic-information/exceptionDetails.ts:73]: Oh good catch!

**Review by bryangaledxw — APPROVED** [review]

**Pull request merged**